frontend/src/app/application-forms/fields/map-url.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { ApplicationFieldsService } from '../_services/application-fields.service';
+
+@Component({
+  selector: 'app-map-url',
+  templateUrl: './map-url.component.html'
+})
+export class MapUrlComponent {
+  @Input() applicantInfo: FormGroup;
+  @Output() blurWebsite: EventEmitter<any> = new EventEmitter();
+
+  constructor(public afs: ApplicationFieldsService) {}
+
+  blurHandler(): void {
+    // function to make blur event bubble up
+    this.blurWebsite.emit();
+  }
+}
```

Comment:
Emitting events and adding listeners can be costly and we should avoid using them if we can. We also want to follow the existing patterns in the codebase as much as possible for code consistency.

Let's remove the blur handler and event emission from the component, and instead leverage the (on change) events that already exist i.e:
```
this.applicationForm.get('applicantInfo.website').valueChanges.subscribe(website => {
    ...
    do the stuff inside matchUrls()
    ...
});
```

Reply:
I've been looking at this, and, honestly I'm not even sure what I was trying to accomplish with this anymore. I removed any use or reference of the blurHandler and noticed no difference in functionality. I must be missing something, but I'm going to move on to your other reviews for now.

Reply:
@Dmac26 once you have made the requested change you can go ahead and hit the `Resolve conversation` button. After you've resolved all the conversations for requested changes you can go ahead and ask for a re-review.